src/languages/bigquery/bigquery.formatter.ts

```diff
@@ -105,7 +105,7 @@ const reservedCommands = expandPhrases([
   'EXPORT DATA',
 ]);
 
-const reservedBinaryCommands = expandPhrases([
+const reservedSetOperations = expandPhrases([
   'UNION {ALL | DISTINCT}',
```

Comment:
should this line be brackets?

Reply:
Yep. BigQuery does not allow plain `UNION`: https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#set_clause